homeassistant/components/ambient_station/__init__.py

```diff
@@ -218,6 +220,8 @@
     TYPE_TEMPINF: ("Inside Temp", TEMP_FAHRENHEIT, TYPE_SENSOR, "temperature"),
     TYPE_TOTALRAININ: ("Lifetime Rain", "in", TYPE_SENSOR, None),
     TYPE_UV: ("uv", "Index", TYPE_SENSOR, None),
+    TYPE_PM25: ("PM25", "μg/m^3", TYPE_SENSOR, None),
```

Comment:
Make sure to use the constant from `const.py`: `CONCENTRATION_MICROGRAMS_PER_CUBIC_METER`